lib/amazon-pricing/definitions/rds-instance-type.rb

```diff
@@ -85,8 +85,10 @@ def update_pricing(database_type, type_of_instance, json, is_multi_az, is_byol)
           end
         end
       end
-      # Copy prices from oracle_se to oracle_se2
+      # Copy byol prices from oracle_se to oracle_{ee,se1,se2}
       if database_type == :oracle_se && is_byol
+        update_pricing :oracle_ee,  type_of_instance, json, is_multi_az, is_byol
```

Comment:
for oracle: we only collect for non-byol, since all byol prices are the same;  so we collect for oracle_se, oracle_se1, and, oracle_se2:
`    @@DB_Deploy_Types = {
      ...
      :oracle_se1   => [:standard, :multiaz, :byol, :byol_multiaz],
      :oracle_se2   => [:standard, :multiaz, :byol, :byol_multiaz],
      :oracle_se    => [:byol, :byol_multiaz],
      :oracle_ee    => [:byol, :byol_multiaz],
`
All of the above byol variants are the same, hence we copy (from oracle_se);  and that's why (for oracle) we don't collect from oracle_ee.  In summary, we collect {se, se1, se2}.